tests/ui/get_unwrap.rs

```diff
@@ -1,5 +1,6 @@
 // run-rustfix
 #![allow(unused_mut)]
+#![deny(get_unwrap)]
```

Comment:
```suggestion
#![deny(clippy::get_unwrap)]
```
That should fix the test failure